project-document/docs/specifications/terms.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Terms
+---
+
+## User
+
+A user in this system is required to register using the Google Account.
+
+* When a user has registered, a new Group and Container for the user will be also created which names are *{user's name}'s group* and *{user's name}'s container* respectively. 
+
+## Container
+
+The Container is a box of foods like a fridge.
+The container must belong to the Group.
+A user who doesn't belong any group **can** create a new container without group, but then the group is created for it automatically.
```

Comment:
@hitohata 
Now, a user cannot create a new container directly without any group; a user has to choose a group before creating a group.

Reply:
@taniguchiiqqq 
Thank you. I've fixed it. (deleted it sentence.)